tests/Security/IssueTriageWorkflowPermissionsTest.php

```
@@ -0,0 +1,151 @@
+    }
+
+    public function testItGrantsNothingBeyondIssuesAndIdToken(): void
+    {
+        $granted = [];
+
+        foreach ($this->lines() as $line) {
+            // `  issues: write`, `      contents: read` — a permission
+            // grant is a bare `key: read|write|none` line. The workflow's
+            // own `permissions: {}` and every other key (`name:`, `on:`)
+            // fail this shape and are ignored.
+            if (preg_match('/^\s+([a-z][a-z-]*):\s*(read|write|none)\s*$/', $line, $m) !== 1) {
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $m. Configured minimum length is 3. (undefined)

(ShortVariable)